extension_templates/forecasting.py

```diff
@@ -90,6 +90,7 @@ class MyForecaster(BaseForecaster):
         "requires-fh-in-fit": True,  # is forecasting horizon already required in fit?
         "X-y-must-have-same-index": True,  # can estimator handle different X/y index?
         "enforce_index_type": None,  # index type that needs to be enforced in X/y
+        "capability:pred_int": False,  # is forecaster probabilistic?
```

Comment:
maybe a better description: "does forecaster implement `predict_quantiles`?"

Also consider renaming this to `"capability:predict_quantiles"`, but perhaps it's best to leave it like this since people may know the tag.